frontend/src/js/services/services.js

```diff
@@ -104,6 +104,53 @@
             angular.element("#sim-loader").fadeOut();
             angular.element(".web-container").removeClass('low-screen');
         };
+
+        this.check = function(password){
+
+           //Regular Expressions.  
+            var regex = new Array();
+            regex.push("[A-Z]"); //Uppercase Alphabet.  
+            regex.push("[a-z]"); //Lowercase Alphabet.  
+            regex.push("[0-9]"); //Digit.  
+            regex.push("[$$!%*#?&]"); //Special Character.  
+            var passed = 0;
+            //Validate for each Regular Expression.  
+            for (var i = 0; i < regex.length; i++) {
+                if (new RegExp(regex[i]).test(password)) {
+                    passed++;
+                }
+            }
+            //Validate for length of Password.  
+            if (passed > 2 && password.length > 5) {
```

Comment:
In signup form minimum length of password is `8` so please change `5` with `8`.

Reply:
then I guess we must remove ng-minlength="8"